### DISCUSSION TITLE
alist发布版本只有30M不到，为什么自己编译有70M左右

### DISCUSSION THREAD

**Discussion by huolanyun** [AlistGo/alist#6506] [General]
按照安稳大佬的编译，不解

**Comment by li-peifeng** [comment]
因为你自己编译的里面包含原本的镜像，编译只会加大体积，不会减小

**Comment by anwen-anyi** [comment]
看截图上方的是使用了upx压缩是27MB，没使用upx压缩的是70MB
![image](https://github.com/alist-org/alist/assets/56105412/f306d272-d276-4893-8834-1e570e36bd69)

**Reply by huolanyun** [reply]
linux编译怎么压缩

**Reply by li-peifeng** [reply]
我用github的actions交叉编译后的都是30来兆啊？

**Reply by huolanyun** [reply]
> 我用github的actions交叉编译后的都是30来兆啊？

不会用

**Reply by li-peifeng** [reply]
安稳的教程里有

**Reply by huolanyun** [reply]
> 安稳的教程里有

https://www.yuque.com/anwenya/alist/glqlhu#dV53r
按照这个教程，编译后71M

**Reply by li-peifeng** [reply]
https://anwen-anyi.github.io/index/build/github.html

用这个

**Reply by huolanyun** [reply]
> https://anwen-anyi.github.io/index/build/github.html
> 
> 用这个

OK，3q

**Comment by anwen-anyi** [comment]
`Linux`编译出来的是压缩的吗？ 只压缩包倒是文件不大，解压出来呢

**Reply by huolanyun** [reply]
https://www.yuque.com/anwenya/alist/glqlhu#dV53r
按照这个教程，编译后exe  71M

**Comment by pongfcnkl** [comment]
可以编译完成后用upx压缩，70m能压到20多m
